src/utils/env.js

```diff
@@ -1,27 +1,34 @@
+// dotenv loading similar to create-react-app / react-scripts or dotenv-load
+
 const path = require('path')
 const dotenv = require('dotenv')
-const { statAsync, readFileAsync } = require('../lib/fs')
+const filterObject = require('filter-obj')
+const { isFileAsync, readFileAsync } = require('../lib/fs')
 
 async function getEnvSettings(projectDir) {
-  const envDevelopmentFile = path.resolve(projectDir, '.env.development')
-  const envFile = path.resolve(projectDir, '.env')
-
-  const settings = {}
-
-  try {
-    if ((await statAsync(envFile)).isFile()) settings.file = envFile
-  } catch (err) {
-    // nothing
-  }
-  try {
-    if ((await statAsync(envDevelopmentFile)).isFile()) settings.file = envDevelopmentFile
```

Comment:
We could have optimised the previous code actually to return if we manage to read `.env.development` and skip reading `.env`, but that's irrelevant now.